### DISCUSSION TITLE
slow chain response

### DISCUSSION THREAD

**Discussion by m4c3** [ipxe/ipxe#247] [Q&A]
hi,
i want to load a ipxe script via http chain from an external server
unfortunately the server does some really slow lookups in a cmdb system. the response takes around 25sec.

i tried to increase the timeout in the chain command
`chain -t 40000 http://myserver/${net0/mac}`
but after 10 seconds the request fails.

is there another way to increase the timeout?

thanks!

**Comment by mcb30** [comment]
What error do you get from the failed `chain` command?

iPXE will always include an error URL in any error message.  These error messages are essential for any kind of problem diagnosis.

**Comment by stappersg** [comment]
> i tried to increase the timeout in the chain command
> `chain -t 40000 http://myserver/${net0/mac}`
> but after 10 seconds the request fails.

Does that say _"Despite forty seconds time out time, there is failure after already ten seconds"_?

@m4c3 What about the _"What error do you get from the failed `chain` command?"_ ?
